### DISCUSSION TITLE
Ability to disable damage from fragments when the walls of the submarine are damaged

### DISCUSSION THREAD

**Discussion by IgorParshukov** [FakeFishGames/Barotrauma#13063] [Ideas and suggestions]
Hello, I would like to add such a feature in the game as with Jupiter radiation, but with this innovation in the game NPCs have become a very big problem. When the submarine's hull is damaged, NPCs run to repair the holes with the whole team, after which they receive further damage from repeated attacks from monsters. Also, missions to transport passengers have become very problematic, since passengers quickly die due to the dullness of their intelligence and they cannot prepare for the next attacks of monsters (which also applies to the crew). This innovation is well suited for multiplayer, but not single player play with bots.

**Comment by SomeRandomNoobKekeke** [comment]
here's a mod that disables damage from hull fragments
https://steamcommunity.com/sharedfiles/filedetails/?id=3090397960
